### ISSUE TITLE
Narrator indicates REPL window as 'text editor readonly'

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS#3175]
Should be instead reading `R Interactive`. Same in History window

**Comment by MikhailArkhipov** [comment]
This will have to be fixed in IW code. C# is aware of the issue. https://github.com/dotnet/interactive-window/issues/37
